docusaurus.config.js

```diff
@@ -177,29 +184,35 @@ module.exports = {
           flexBasis: '80px',
         },
       },
-    ]
+    ],
   },
   stylesheets: [
-    'https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,700;1,400&family=Roboto:wght@900&display=swap'
+    'https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,700;1,400&family=Roboto:wght@900&display=swap',
   ],
   themes: [
-    ['@docusaurus/theme-classic', {
-      customCss: require.resolve('./src/css/docs.css')
-    }],
-    ['@docusaurus/theme-search-algolia', {}]
+    [
+      '@docusaurus/theme-classic',
+      {
+        customCss: require.resolve('./src/css/docs.css'),
+      },
+    ],
+    ['@docusaurus/theme-search-algolia', {}],
   ],
   plugins: [
-    ['@docusaurus/plugin-content-docs', {
-      path: 'docs',
-      sidebarPath: require.resolve('./sidebars.json'),
-      // This is a holdover because we set up original docusaurus
-      // to support translations and v2 doesn't support them yet
-      routeBasePath: 'docs/en/',
-    }],
-    "docusaurus-plugin-sass",
+    [
+      '@docusaurus/plugin-content-docs',
+      {
+        path: 'docs',
+        sidebarPath: require.resolve('./sidebars.json'),
+        // This is a holdover because we set up original docusaurus
+        // to support translations and v2 doesn't support them yet
+        routeBasePath: 'docs/en/',
+      },
+    ],
+    'docusaurus-plugin-sass',
     ['@docusaurus/plugin-content-pages', {}],
-    isProd && ['@docusaurus/plugin-google-analytics', {}],
-    isProd && ['@docusaurus/plugin-google-gtag', {}],
-    isProd && ['@docusaurus/plugin-sitemap', {}],
-  ]
+    ['@docusaurus/plugin-google-analytics', {}],
+    ['@docusaurus/plugin-google-gtag', {}],
+    ['@docusaurus/plugin-sitemap', {}],
```

Comment:
Version `alpha.68` and higher fixed this issue (with https://github.com/facebook/docusaurus/pull/3704), so we can add these back.
```suggestion
    isProd && ['@docusaurus/plugin-google-analytics', {}],
    isProd && ['@docusaurus/plugin-google-gtag', {}],
    isProd && ['@docusaurus/plugin-sitemap', {}],
```